src/DataValues/DecimalValue.php

```diff
@@ -353,6 +353,24 @@ public static function newFromArray( $data ) {
 	}
 
 	/**
+	 * Create DecimalValue from regular numeric string or value.
+	 * @param int|float|string $number
+	 *
+	 * @return self
+	 */
+	public static function makeDecimalValue( $number ) {
+
+		if ( is_string( $number ) && $number !== '' ) {
+			if ( $number[0] !== '-' && $number[0] !== '+' ) {
+				$number = '+' . $number;
+			}
+		}
+
+		return new static( $number );
+	}
```

Comment:
I'd be ok with either way. The main thing is ability to accept string like "1234".